### ISSUE TITLE
Example with R3F (React-Three-Fiber) ? 

### ISSUE THREAD

**Issue by Lorenzowy** [vasturiano/3d-force-graph#412]
**Is your feature request related to a problem? Please describe.**
Is it possible to use R3F with your package? 
Could be
- 3d-force-graph within an R3F canvas 
- R3F component embedded into 3d-force-graph 

**Describe the solution you'd like**
Provide an example if either is possible?

**Describe alternatives you've considered**
I'm able to code with normal threeJS, but was wondering if r3f was possible.

**Comment by vasturiano** [comment]
@Lorenzowy I don't think you could integrate with this component directly since both have their own scene and renderer.

But you could possibly use it with [three-forcegraph](https://github.com/vasturiano/three-forcegraph). I suppose you'd just need to make a React wrapper around that ThreeJS plain component that could be included in R3F.
